itests/f3_test.go

```diff
@@ -275,14 +326,16 @@ func setupWithStaticManifest(t *testing.T, manif *manifest.Manifest, testBootstr
 		err = n.NetConnect(ctx, e.ms.PeerInfo())
 		require.NoError(t, err)
 	}
-
 	errgrp.Go(func() error {
 		defer func() {
 			require.NoError(t, manifestServerHost.Close())
 		}()
 		return e.ms.Run(ctx)
 	})
 
+	// Assure manifest is picked up by all nodes.
+	e.ms.UpdateManifest(manif)
+	e.waitTillManifestChange(manif, 20*time.Second)
```

Comment:
This shouldn't be necessary. All nodes are constructed with `manif` as their initial manifest.

Reply:
I thought I assert it anyway for sanity. Happy to remove.

Reply:
I'd kind of prefer to test it without it.